### PR TITLE
Add recruitment page

### PR DESCRIPTION
**Pull request by Dillonsd**
- Adds a link to the recruitment page to the header
- Adds the recruitment page design as per the Figma design
- Adds links to the Google forms
- Adds a subteam interface (could be beneficial for when the backend is implemented)

**Waiting on the descriptions for the subteam applications**
**Waiting on assets for the page**

### CODE REVIEW

**Review by JessieG123 — CHANGES_REQUESTED** [review]
This picture of Elysia is a bit off screen is that suppose to be like that? 
image

**Comment by Dillonsd** [comment]
> This picture of Elysia is a bit off screen is that suppose to be like that?
> image

That's cz it's a placeholder image.

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Here's the rest.

Communications: https://forms.gle/w6YJ4akBfVYJ8oGs9 

Multi-team: https://forms.gle/zwn3Tk7Pgf89648a6

**Review by JessieG123 — COMMENTED** [review]
I think if these are left aligned it might look better or have some space between the text and the edge of the right screen. 
image

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Looks awesome! Just one last thing

Seems to be a lot of white space on the right. Is it possible to decrease it a little so that the text and pic are relatively centered?
image

**Review by JaydenBischoff — CHANGES_REQUESTED** [review]
Looks great! I have a few things that need to change:

- The info nights are actually the 13th and 16th at 6:30pm
- I didn't anticipate the red and white photo being used. I think it looks nice but I don't like how it says the same thing as the text above. Could you either replace the text at the top with "recruiting now", or replace the image with the other one that I just added to figma, or if you have another idea go for it. 
- Could you also make the image of the car at the top right be bigger/ spread out a bit more
- Last thing, could you change the link on the main page so that it links to this page

Thanks!

**Review by JessieG123 — CHANGES_REQUESTED** [review]
Looks great! 

Just one thing, on a bigger screen the Recruiting Now is at the very edge of the screen. If it's not too much maybe add some space?
image

File: src/app/business/recruitment/recruitment.component.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    <div class="infoContainer">
+        <div>
+            <h1 class="infoTitle">Information Nights</h1>
+            <p class="infoText">
+                September 13, 2021<br />
+                6:30 pm<br />
+                ENGG Lounge
+            </p>
+            <p class="infoText">
+                September 17, 2021<br />
```
Comment (JessieG123) [inline src/app/business/recruitment/recruitment.component.html:24]: Should be September 16 for the second info night.

**Review by JessieG123 — CHANGES_REQUESTED** [review]

**Comment by Dillonsd** [comment]
> Looks great!
> 
> Just one thing, on a bigger screen the Recruiting Now is at the very edge of the screen. If it's not too much maybe add some space?
> image

Hm, what size screen is this? There should be padding

**Comment by JaydenBischoff** [comment]
Hmm the text seems to get cut off for me. Maybe make it a bit smaller or put the text and picture in a grid container.
image

**Comment by JessieG123** [comment]
> > Looks great!
> > Just one thing, on a bigger screen the Recruiting Now is at the very edge of the screen. If it's not too much maybe add some space?
> > image
> 
> Hm, what size screen is this? There should be padding

1920 x 1080

**Review by JessieG123 — APPROVED** [review]
Excellent. Looks good!

**Review by JaydenBischoff — APPROVED** [review]
I'm still seeing the text issue but it's a bit better. Lets merge it for now and look into fixing it as another task.

**Pull request merged**